tests/util.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass
+from pathlib import Path
+
+from git import Actor, Repo
+
+SOME_COMMIT_MESSAGE = "some-commit-message"
+SOME_ACTOR = Actor("some-user", "some-user@wayfair.com")
+SOME_CONTENT = "some-content"
+
+
+@dataclass
+class Repos:
+    local_repo: Repo
+    remote_repo: Repo
+
+
+def _get_local_remote_repos(base_path: Path) -> Repos:
+    remote_repo_path = base_path / "some-remote-repo"
+    local_repo_path = base_path / "some-local-repo"
+    remote_repo = Repo.init(remote_repo_path)
+
+    # remote repo must have initial content in order to be cloned
+    new_file_path = remote_repo_path / "foo.txt"
+    new_file_path.write_text("some changes")
+    remote_repo.index.add([str(new_file_path)])
+    remote_repo.index.commit(
+        SOME_COMMIT_MESSAGE, author=SOME_ACTOR, committer=SOME_ACTOR
+    )
+
+    local_repo = Repo.clone_from(remote_repo.working_dir, local_repo_path)
+    return Repos(local_repo=local_repo, remote_repo=remote_repo)
+
+
+def _get_diff(repo: Repo) -> str:
```

Comment:
this function feels very helpful - thank you for writing it!

Reply:
thought: a `diff` is just a general comparison between two states, so it isnt immediately clear that this function will get the difference between the latest commit and and unstaged / untracked changes. 

could we either rename this function to indicate this constraint / intention, or include a comment explaining this?